### ISSUE TITLE
Easy to accidentally click a user profile or community link instead of post link in the main view

### ISSUE THREAD

**Issue by a1studmuffin** [dessalines/jerboa#488]
** Jerboa Version **
0.32

**Describe the bug**
In the Card/Small Card/List post views, it's easy to accidentally open the wrong link (user profile or community) instead of the comments.

![image](https://github.com/dessalines/jerboa/assets/6295625/d6d6b3bf-0d46-43b1-bd2a-c8598a6f838c)

![image](https://github.com/dessalines/jerboa/assets/6295625/19c00568-ce3f-4fb3-97ca-7ad325e80561)

**To Reproduce**

1. Click in the wrong spot :)

**Proposed fix**
Either change default behaviour or add a "fat finger/simple links" setting, so the highlighted elements open the post comments (like most other areas on the card) instead of the user profile/community links. The action bar for posts could be extended to add "Go to User Profile" and "Go to Community" accordingly, so these links are still reachable.

I'm happy to put together a PR for this if there's agreement.

**Comment by oscarnylander** [comment]
I've noticed this as well.

What RIF does is to allow navigation to the submitting user using a Context Menu, accessed by long pressing on the list-item. This might be a good way to do it for this project too.

**Comment by maltfield** [comment]
Please don't make the solution to this be making the layout less compact. An optional "fat finger" option is fine, but users who choose the list view choose it because **they *want* a compact view**

**Comment by oscarnylander** [comment]
> Please don't make the solution to this be making the layout less compact. An optional "fat finger" option is fine, but users who choose the list view choose it because **they *want* a compact view**

I think it's fine to keep the layout as compact as it is today, but simply to try to pack fewer touch targets within this compact layout.

Fwiw, Android platform guidelines recommend that touch targets are at least 48dp x 48dp, with a 8dp separation.

**Comment by twizmwazin** [comment]
If we wanted to follow boost, we should add an option to disable all of the small click targets in the post list view. For example, if you want to go to the user of a post, either click the post and then click on the user name within the post view, or press the overflow menu button and select "go to user'.

**Comment by oscarnylander** [comment]
> If we wanted to follow boost, we should add an option to disable all of the small click targets in the post list view. For example, if you want to go to the user of a post, either click the post and then click on the user name within the post view, or press the overflow menu button and select "go to user'.

That sounds like a reasonable compromise.

**Comment by a1studmuffin** [comment]
Yep, that is exactly what I was trying to get at with my proposed fix but evidently didn't communicate the idea properly - no layout changes, just removing small click links.

**Comment by abluescarab** [comment]
> I've noticed this as well.
> 
> What RIF does is to allow navigation to the submitting user using a Context Menu, accessed by long pressing on the list-item. This might be a good way to do it for this project too.

Bump on this one. Definitely my favorite implementation of the feature. I rarely want to visit a user/community more than I just want to see the post.

**Comment by jordan1776** [comment]
> Yep, that is exactly what I was trying to get at with my proposed fix but evidently didn't communicate the idea properly - no layout changes, just removing small click links.

Really everything that isnt view comments or view link should not be a link on the list post view. I like Relay's implementation which is gesturing left on the post (or long pressing) shows the options directly on the space of the post, as if the options were just off screen on the right side. An action bar would work too (hidden by default).

As a side note I think the target for going to the comments should be expanded to everything between the seperator bars, like in card view. When long pressing posts with links now, the box doesn't go all the way to the right edge.
